extensions/http-curl/client/HTTPClient.cpp

```diff
@@ -120,8 +120,21 @@ void HTTPClient::forceClose() {
 
 }
 
+int HTTPClient::debug_callback(CURL *handle, curl_infotype type, char *data, size_t size, void *userptr) {
```

Comment:
I think its much better to log this properly when setVerbose is used, instead of it just going to stderr, but if there is a reason for that I would be willing to discuss this.

Reply:
@phrocker That is a good idea, but for maintaining API-compatibility we will have to choose a default value for that flag. I prefer the default value to be the logger mode. What do you think?